src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -783,7 +783,7 @@ namespace cryptonote
     if(bvc.m_added_to_main_chain)
     {
       cryptonote_connection_context exclude_context = boost::value_initialized<cryptonote_connection_context>();
-      NOTIFY_NEW_BLOCK::request arg = AUTO_VAL_INIT(arg);
+      NOTIFY_NEW_FLUFFY_BLOCK::request arg = AUTO_VAL_INIT(arg);
```

Comment:
That will partition the network. I don't think there's a P2P version broadcast on connect, but if not one could be added, and use the new fluffy types only when the peer is recent enough. Partitioning in this way would be very harsh to the network I think, since the "good" chain would have ~0 hash rate for a good while.